fbw-a380x/src/systems/instruments/src/ND/instrument.tsx

```diff
@@ -167,10 +174,10 @@ const original = document.createElement.bind(document);
 const extraSvgTags = ['tspan'];
 
 document.createElement = ((tagName, options) => {
-    if (extraSvgTags.includes(tagName)) {
-        return document.createElementNS('http://www.w3.org/2000/svg', tagName, options);
-    }
-    return original(tagName, options);
+  if (extraSvgTags.includes(tagName)) {
+    return document.createElementNS('http://www.w3.org/2000/svg', tagName, options);
+  }
+  return original(tagName, options);
```

Comment:
This hack is no longer required. The avionics framework supports tspan now.

Reply:
roger, deleted